Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -82,32 +82,56 @@ FieldProbe::FieldProbe (std::string rd_name)
     if (m_probe_geometry_str == "Point")
     {
         m_probe_geometry = DetectorGeometry::Point;
+        x_probe = 0._rt;
+        y_probe = 0._rt;
+#if !defined(WARPX_DIM_1D_Z)
         getWithParser(pp_rd_name, "x_probe", x_probe);
+#endif
+#if defined(WARPX_DIM_3D)
```

Comment:
We also support RZ with this probe.
There, particles have 3D positions. I think we need these arguments then?

Reply:
I don't think we're supporting RZ yet, which is why I didn't cover this case.

https://github.com/ECP-WarpX/WarpX/blob/5692067b01252146b8d0a90137a1bd952644c8ca/Source/Diagnostics/ReducedDiags/FieldProbe.cpp#L50-L54

But if you want, I can update this PR to take RZ geometry into account, anticipating that FieldProbe + RZ will work in the future.

Reply:
Oh, that one is outdated since we added the particle container :-o

Let's add an RZ test, because this should really just work since #2430

Reply:
@TiberiusRheaume can you please take this on in a follow-up?
```suggestion
#if defined(WARPX_DIM_3D) || defined(WARPX_DIM_RZ)
```

Add the marked defines and run the LWFA (laser-driven e- acceleration) RZ case?
https://warpx.readthedocs.io/en/latest/usage/examples.html#laser-driven-electron-acceleration